### PR TITLE
add update URL

### PR DESCRIPTION
**Pull request by clouserw**
Once this lands, can you give me a new signed add-on and then I can upload that to s3?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "repository": "meandavejustice/min-vid",
   "main": "index.js",
   "author": "meandave",
+  "updateURL": "https://testpilot.firefox.com/files/min-vid/updates.json",
```
Comment (meandavejustice) [inline package.json]: `updateURL` needs to be `path/to/update.rdf`
[example from TxP addon](https://github.com/mozilla/testpilot/blob/master/addon/package.json#L19)

**Comment by meandavejustice** [comment]
closing for #171

File: package.json (hunk 1 above)
Comment (wresuolc) [inline package.json]: rdf is old and busted.  .json is new and hot.  Also, please get rid of the updateLink you added - that'll come from the json (and I'll add a checksum to it).

**Comment by wresuolc** [comment]
merging this one doesn't matter, but reopening so we don't lose track of it

**Comment by meandavejustice** [comment]
closing for #174 

- fixed updateURL to point to `updates.json`
- remove call to update-version bin script
- remove handling of `update.rdf` from package.json
- bump version

**Pull request closed without merging**